lib/ApiRequestor.php

```diff
@@ -254,9 +254,8 @@ private function _processResourceParam($resource, $hasCurlFile)
 
     private function _interpretResponse($rbody, $rcode, $rheaders)
     {
-        try {
-            $resp = json_decode($rbody, true);
-        } catch (Exception $e) {
+        $resp = json_decode($rbody, true);
+        if ($resp === null) {
```

Comment:
We should use [json-last-error()](http://www.php.net/manual/en/function.json-last-error.php) and/or [json-last-error-msg](http://php.net/manual/en/function.json-last-error-msg.php) at least in our logs to help our users debug. Flagging here instead of in private to avoid merging until then.